packages/contracts/contracts/L1/DepositFeed.sol

```diff
@@ -26,6 +26,14 @@ contract DepositFeed {
         bytes data
     );
 
+    /**
+     * Accepts value deposits where users can send ETH directly to
+     * the smart contract address and have the funds be deposited
+     */
+    receive() external payable {
+        depositTransaction(msg.sender, msg.value, 30000, false, bytes(""));
```

Comment:
Is 30k the intrinsic gas or should it be lower?
Someone else will have to verify that gas works / there is not trickery that can be done here & this can be integrated with deposit fees.

Reply:
Isn't an ETH transfer only 21k?

Reply:
To be honest, I just picked a number here than I was sure would be high enough. I feel like we want a buffer over the minimal 21k and an e2e test to make sure that the number works with L2 geth

Reply:
I like this change, I think the `receive()` function is a definite UX improvement, but...

> and an e2e test to make sure that the number works with L2 geth

If we think this would be a useful test to have, then IMO we should add it in this PR.

Reply:
That could be done fairly easily building on #269

Reply:
We can also make this part of the e2e go tests.